incubator-tools/amount_rectification_from_words_to_numbers/amount_rectification_from_words_to_numbers.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #470.                output_path.split("/")[2],
</pre><p>There's a lot of this splitting happening in this notebook. This should really be done in a utility function and preferably as few times as possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
We are using the existing utility functions where we provide bucket and prefix separately. So, in this case i have splitted from the URI and used the existing functions in utilities